wandb/cli/cli.py

```diff
@@ -736,6 +736,25 @@ def _parse_settings(settings):
         entity = parts.get("entity") or entity
         project = parts.get("project") or project
         sweep_id = parts.get("name") or update
+
+        has_project = (project or api.settings("project")) is not None
+        has_entity = entity or api.settings("entity") is not None
```

Comment:
i assume this is meant to be parenthesized like has_project

Reply:
whoops, thanks